distributed/platform.py

```diff
@@ -0,0 +1,70 @@
+import os
```

Comment:
Do we intend this module to be public? If so, add these to the API docs?

Reply:
No, I think this should be just another internal module. Kept it out of `utils.py`, as that module is huge.

Reply:
Then do we want to prefix with an underscore?

Reply:
We don't do this for utils, etc..., so I don't think that's needed here.

Reply:
Right, but was that a deliberate decision? I don't want to make the situation "worse" if we aren't making any API guarantees about these methods.

Reply:
Yeah, I don't personally think that we should underscore all of the non-user-facing modules.  I think that we just shouldn't include them in `__init__.py` and not publicize them too broadly.  If people are reaching into our internal module structure I personally don't mind burning them.

Reply:
OK. I'll likely open up a PR at least documenting that if they're not part of the publicly documented API then they're considered internal.